Model/DataObject/ClassDefinition/Data/ObjectBridge.php

```diff
@@ -957,6 +956,13 @@ private function setFieldDefinition($fieldName, $fd, $field, $readOnly, $hidden)
         if ($fd instanceof ClassDefinition\Data\Select) {
             $this->$fieldName[ $field ]['options'] = $fd->getOptions();
         }
+
+        // Hreftypeahead has showTrigger
+        if ($fd instanceof \PimcoreHrefTypeaheadBundle\Model\DataObject\Data\HrefTypeahead) {
+            $this->$fieldName[ $field ]['showTrigger'] = $fd->getShowTrigger();
+        }
+
```

Comment:
Remove superfluous blank lines

Reply:
@Sunohara666 could you remove these 2 lines? Then it can be merged.

Reply:
This has been done